JASP-Engine/JASP/R/descriptives.R

```diff
@@ -196,6 +196,24 @@ Descriptives <- function(jaspResults, dataset, options) {
     .descriptivesScatterPlots(jaspResults[["scatterPlots"]], dataset.factors, variables, splitName, options)
   }
   return()
+
+   # Interval plots
+   if (options$descriptivesIntervalPlot) {
+     if(is.null(jaspResults[["IntervalPlots"]])) {
+       jaspResults[["IntervalPlots"]] <- createJaspContainer(gettext("Intervalplots"))
+	   jaspResults[["IntervalPlots"]]$dependOn(c("descriptivesIntervalPlot", "splitby"))
+       jaspResults[["IntervalPlots"]]$position <- 11
+     }
+
+  intervalPlots <- jaspResults[["intervalPlots"]]
+
+  for (var in variables) {
+    if(is.null(intervalPlots[[var]]) && .descriptivesIsNumericColumn(dataset.factors, var)) {
+	intervalPlots[[var]] <- .descriptivesIntervalPlot(dataset = dataset, options = options, variable = var)
+      intervalPlots[[var]]$dependOn(optionContainsValue=list(variables=var))
+    }
+  }
+  }
```

Comment:
So the top line (which you didn't edit) is a return statement. That means that no code beyond this point is ever run, which explains why there are no interval plots in the output when you built JASP.